build/e2e.yml

```diff
@@ -24,7 +24,7 @@ resources:
     type: github
     endpoint: github
     name: microsoft/qdk
-    ref: refs/heads/main
+    ref: refs/heads/imiles/task-36357
```

Comment:
This looks like it should be changed back to `main`?

Reply:
I was referencing my separate branch of the qdk which is also using containers for the e2e builds. Once this PR looks good I will change this back to main.